_includes/aip-nav.html

```diff
@@ -40,6 +40,7 @@
         {% endif -%}
 
         {% for category in categories -%}
+          {% if category[menu.front_matter].hidden != true %}
```

Comment:
Added `hidden` since neither of them have an AIP yet and I did not want clutter in the interim.